go-controller/pkg/ovn/ipallocator/allocator/bitmap_test.go

```diff
@@ -155,3 +155,58 @@ func TestContiguousAllocation(t *testing.T) {
 		t.Errorf("unexpected success")
 	}
 }
+
```

Comment:
I think that we should replace s/NewAllocationMap/NewRoundRobinAllocationMap/ in some/all the tests in this file, at least if we are only using this new AllocationMap

Reply:
@aojea I don't think we should replace (to keep the file as similar as possible with upstream Kube) but I'll certainly add duplicate tests for the round-robin one.

Reply:
I wanted to afford you that pain :upside_down_face: